src/com/sap/piper/JenkinsController.groovy

```diff
@@ -0,0 +1,75 @@
+package com.sap.piper
+
+class JenkinsController implements Serializable {
+    def script
```

Comment:
`script` is not set. Of course that variable (and also the other variables) can be set via tuple constructor approach or otherwise in groovy or also after the instance has been created. But if that does not happen the instance cannot be used properly. IMO it is a disdavantage of the tuple constructor / named-parameter approach in groovy since it is easier to make such kind of mistakes.

I would like to propose to introduce a good old Constructor ensuring all required variables are set during the instance is created.

Reply:
done. @fwilhe right?